lib/jobs/ScheduleBackupJob.js

```diff
@@ -18,43 +21,50 @@ const ScheduleManager = require('./ScheduleManager');
 //Intentionally Jobs are kept as static as we register same definition for all Jobs of similar type
 //Any instance specific references if reqiured for any jobs must be kept within run method for any future jobs
 class ScheduleBackupJob extends BaseJob {
-  constructor() {
-    super();
-  }
 
   static run(job, done) {
-    job.__started_At = new Date();
-    const options = job.attrs.data;
-    logger.info(`-> Starting ScheduleBackupJob -  name: ${job.attrs.name} - with options: ${JSON.stringify(options)} `);
-    if (!_.get(options, 'instance_id') || !_.get(options, 'type')) {
-      const msg = `Scheduled backup cannot be initiated as the required mandatory params (intance_uid | type) is empty : ${JSON.stringify(options)}`;
-      logger.error(msg);
-      this.runFailed(new errors.BadRequest(msg), undefined, job, done);
-      return;
-    }
-    const backupRunStatus = {
-      start_backup_status: 'failed',
-      delete_backup_status: 'failed'
-    };
-    let instanceDeleted = false;
-    return this
-      .isServiceInstanceDeleted(options.instance_id)
-      .tap(deleteStatus => instanceDeleted = deleteStatus)
-      .then(() => {
-        if (instanceDeleted) {
-          return 'instance_deleted';
+    return Promise.try(() => {
+        job.__started_At = new Date();
+        const jobData = job.attrs.data;
+        logger.info(`-> Starting ScheduleBackupJob -  name: ${job.attrs.data[CONST.JOB_NAME_ATTRIB]} - with options: ${JSON.stringify(jobData)} `);
```

Comment:
you can make use of jobData instead of job.attrs.data.

Reply:
Even though theres nothing wrong in the above, changing it :-)